docs/content/ActionMenu.mdx

```diff
@@ -0,0 +1,74 @@
+---
+title: ActionMenu
+---
+
+An `ActionMenu` is an ActionList-based component for creating a menu of actions that expands through a trigger button.
+
+## Default example
+
+```jsx live
+<ActionMenu
+  anchorContent="Menu"
+  onAction={({text}) => console.log(text)}
+  items={[
+    {text: 'New file'},
+    ActionMenu.Divider,
+    {text: 'Copy link'},
+    {text: 'Edit file'},
+    {text: 'Delete file', variant: 'danger'}
+  ]}
+/>
+```
+
+## Example with grouped items
+
+```jsx live
+<ActionMenu
+  anchorContent="Menu"
+  onAction={({text}) => console.log(text)}
+  groupMetadata={[
+    {groupId: '0'},
+    {groupId: '1', header: {title: 'Live query', variant: 'subtle'}},
+    {groupId: '2', header: {title: 'Layout', variant: 'subtle'}},
+    {groupId: '3'},
+    {groupId: '4'}
+  ]}
+  items={[
+    {leadingVisual: TypographyIcon, text: 'Rename', groupId: '0'},
+    {leadingVisual: VersionsIcon, text: 'Duplicate', groupId: '0'},
+    {leadingVisual: SearchIcon, text: 'repo:github/memex,github/github', groupId: '1'},
```

Comment:
This should be cleaned up in ActionList as well, noticed it showed up a few places in examples recently
```suggestion
    {leadingVisual: SearchIcon, text: 'repo:github/github', groupId: '1'},
```